server/src/main/java/dev/slimevr/platform/SteamVRBridge.java

```diff
@@ -153,143 +154,177 @@ protected void writeBatteryUpdate(Tracker localTracker) {
-						allTrackers,
-						TrackerPosition.LEFT_LOWER_ARM
+			}
+			case LEFT_ELBOW -> {
+				batteryTrackers
+					.add(
+						TrackerUtils
+							.getNonInternalTrackerForBodyPosition(
+								allTrackers,
+								TrackerPosition.LEFT_LOWER_ARM
+							)
 					);
-				tertiaryTracker = TrackerUtils
-					.getNonInternalTrackerForBodyPosition(
-						allTrackers,
-						TrackerPosition.LEFT_SHOULDER
+				batteryTrackers
+					.add(
+						TrackerUtils
+							.getNonInternalTrackerForBodyPosition(
+								allTrackers,
+								TrackerPosition.LEFT_SHOULDER
+							)
 					);
-				break;
-			case RIGHT_ELBOW:
-				secondaryTracker = TrackerUtils
-					.getNonInternalTrackerForBodyPosition(
-						allTrackers,
-						TrackerPosition.RIGHT_LOWER_ARM
+			}
+			case RIGHT_ELBOW -> {
+				batteryTrackers
+					.add(
+						TrackerUtils
+							.getNonInternalTrackerForBodyPosition(
+								allTrackers,
+								TrackerPosition.RIGHT_LOWER_ARM
+							)
 					);
-				tertiaryTracker = TrackerUtils
-					.getNonInternalTrackerForBodyPosition(
-						allTrackers,
-						TrackerPosition.RIGHT_SHOULDER
+				batteryTrackers
+					.add(
+						TrackerUtils
+							.getNonInternalTrackerForBodyPosition(
+								allTrackers,
+								TrackerPosition.RIGHT_SHOULDER
+							)
 					);
-				break;
+			}
 		}
 
 		// If the battery level of the tracker is lower than lowestLevel, then
 		// the battery level of the tracker position becomes lowestLevel.
 		// Tracker voltage is set if the tracker position has a battery level
-		// lower than lowest level and has a battery voltage (owoTrack devices
-		// do not).
-		if (
-			(primaryTracker != null) && (primaryTracker.getBatteryLevel() != null)
-		) {
-			lowestLevel = primaryTracker.getBatteryLevel();
-
-			if (primaryTracker.getBatteryVoltage() != null) {
-				trackerVoltage = primaryTracker.getBatteryVoltage();
-			}
-		}
-		if (
-			(secondaryTracker != null)
-				&& (secondaryTracker.getBatteryLevel() != null)
-				&& (secondaryTracker.getBatteryLevel() < lowestLevel)
-		) {
-			lowestLevel = secondaryTracker.getBatteryLevel();
-
-			if (secondaryTracker.getBatteryVoltage() != null) {
-				trackerVoltage = secondaryTracker.getBatteryVoltage();
-			}
-		}
-		if (
-			(tertiaryTracker != null)
-				&& (tertiaryTracker.getBatteryLevel() != null)
-				&& (tertiaryTracker.getBatteryLevel() < lowestLevel)
-		) {
-			lowestLevel = tertiaryTracker.getBatteryLevel();
-
-			if (tertiaryTracker.getBatteryVoltage() != null) {
-				trackerVoltage = tertiaryTracker.getBatteryVoltage();
+		// lower than the lowest level and has a battery voltage (owoTrack
+		// devices do not).
+		for (Tracker batteryTracker : batteryTrackers) {
+			if (batteryTracker != null && batteryTracker.getBatteryLevel() != null) {
+				lowestLevel = batteryTracker.getBatteryLevel();
+
+				if (batteryTracker.getBatteryVoltage() != null) {
+					trackerVoltage = batteryTracker.getBatteryVoltage();
+				}
 			}
```

Comment:
`lowestLevel` is no longer checked here, so instead of using the lowest battery level, this PR makes it use the last tracker's battery level. Should be fixed by just adding `batteryTracker.getBatteryLevel() < lowestLevel` to the if condition. I'd make a suggestion but formatting would be a little messed up.

Reply:
I cannot believe no one caught this